### PR TITLE
chore(ci): include snippets folder into tfhe npm package (backport 0.7.x)

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -414,6 +414,7 @@ build_web_js_api_parallel: install_rs_check_toolchain install_wasm_pack
 		wasm-pack build --release --target=web \
 		-- --features=boolean-client-js-wasm-api,shortint-client-js-wasm-api,integer-client-js-wasm-api,parallel-wasm-api,zk-pok \
 		-Z build-std=panic_abort,std
+	jq '.files += ["snippets"]' tfhe/pkg/package.json > tmp_pkg.json && mv -f tmp_pkg.json tfhe/pkg/package.json
```
Comment (IceTDrinker) [inline Makefile:417]: alignment isi not good, I think it breaks Makefiles

File: ci/slab.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [backend.aws.cpu-big]
 region = "eu-west-3"
-image_id = "ami-051942e4055555752"
+image_id = "ami-09b5f2f71828035d4"
```
Comment (IceTDrinker) [inline ci/slab.toml:3]: correct AMI ?

File: Makefile (hunk 1 above)
Comment (soonum) [reply Makefile:417]: It's ok, since it's not on the same line of command than the previous one.

**Review by IceTDrinker — APPROVED** [review]

**Comment by IceTDrinker** [comment]
wasm tests are green, looking good

**Pull request merged**